### ISSUE TITLE
config.example.yaml has data source in the wrong keyspace.

### ISSUE THREAD

**Issue by zfletcher** [quay/clair#204]
It fails to run unless it is in the database keyspace:

```
$ docker run --rm -ti -p 6060-6061:6060-6061 -v $PWD/config:/config -v /tmp:/tmp --link clair_postgres --net clair_default quay.io/coreos/clair -config /config/config.yaml
2016-06-14 18:10:02.017211 C | main: failed to load configuration: could not load configuration: no database source specified
```

My patch fixes clair config and now it runs as expected.

https://github.com/zfletcher/clair/commit/c9a2272a94994e7f2bd5b3d16c6f7ff0f9ea3384

```
$ docker logs -f clair_clair
2016-06-14 18:18:17.216011 I | pgsql: running database migrations
goose: no migrations to run. current version: 20151222113213
2016-06-14 18:18:17.294032 I | pgsql: database migration ran successfully
2016-06-14 18:18:17.294260 I | notifier: notifier service is disabled
2016-06-14 18:18:17.294345 I | api: starting main API on port 6060.
2016-06-14 18:18:17.294577 I | api: starting health API on port 6061.
2016-06-14 18:18:17.294685 I | updater: updater service started. lock identifier: 8ac5f141-e6bc-425c-b3ec-592e3f139fb3
2016-06-14 18:18:17.300814 I | updater: updating vulnerabilities
2016-06-14 18:18:17.300918 I | updater: fetching vulnerability updates
2016-06-14 18:18:17.300971 I | updater/fetchers/ubuntu: fetching Ubuntu vulnerabilities
2016-06-14 18:18:17.310008 I | updater/fetchers/debian: fetching Debian vulnerabilities
2016-06-14 18:18:17.310521 I | updater/fetchers/rhel: fetching Red Hat vulnerabilities
2016-06-14 18:28:07.318672 I | updater: adding metadata to vulnerabilities
2016-06-14 18:45:27.597493 I | updater: update finished
```

Please advise if this is a real issue, and if so, do you need me to PR it... ??

**Comment by jgsqware** [comment]
As said in #203, the config.example.yml refer to the master branch, where the configuration is correct. Your patch refer to the release.

@jzelinskie @Quentin-M, we should address this point, we receive a lot of question around that. Any idea?

**Comment by zfletcher** [comment]
Sorry!!  I am using the official image on quay.io/coreos, but was following the Master Readme.. My apologies for the erroneous question.

**Comment by jgsqware** [comment]
It's a known mistake and we should address that to avoid confusion

**Comment by Quentin-M** [comment]
As far as I know, the README matches with `quay.io/coreos/clair:latest`, right?

**Comment by zfletcher** [comment]
@Quentin-M Only release-1.2's readme, not master (which now uses options), which hasn't yet been cut into a new release.
So the readmes are right, it is just that folks downloading from docker hub are getting release-1.2 not master, which is intended, but perhaps not well known/realized by consumers.

**Comment by Quentin-M** [comment]
@zfletcher thanks for the heads-up. We did change the way our images are stored on Quay.io. The `latest` tag points to v1.2.2. To get the image built on the `master` branch of this repository, `quay.io/repository/coreos/clair-git` should be pulled instead.

The issue is that in the Quick start guide from the README, the `latest` tag is pulled (currently Clair v1.2.2) but `config.example.yaml` is curl'd from `master`. Let's fix everything to `v1.2.2` in the README as a first step. Maybe we could mention the clair and clair-git image repositories as well somehow.

cc @cynipe
